### ISSUE TITLE
Details about RandLA-Net implementations

### ISSUE THREAD

**Issue by garryz94** [QingyongHu/RandLA-Net#89]
Hi @QingyongHu , thanks for your great work!
I have some questions about your code

1. In the function `relative_pos_encoding`, you use ` tf.reduce_sum(relative_xyz, axis=-1, keepdims=True)` to get `relative_dis`.  Why do you not use euclidean distance, for example  tf.reduce_sum(tf.square(relative_xyz), axis=-1, keepdims=True)?
2. When building LocSE block, you use conv2d as mlp. Is it a better idea to use conv1d? I think  conv1d can eliminate the effect of `num_points`, in order to focus on learning weights on the other two dimension.
Thanks again for your excellent work!
